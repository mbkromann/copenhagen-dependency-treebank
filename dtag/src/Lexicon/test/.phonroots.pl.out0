flyv|fløj = flyv|fløj = [flyv] [^ø, v/j] [^ø, v/j, et] 
lav = lav = [lav] [er] [er, r/s] 
lyt|lytt = lyt|lytt = [lyt] [~, er] [~, ede] 
mand|mænd = mand|mænd = [mand] [^æ] [en] 
pukkel|pukl = pukkel|pukl = [pukkel] [-~e, er] [-~e, er, ne] 
tal|tæl = tal|tæl = [tæl] [^a, te] [^a, t] 
tidsel|tidsl = tidsel|tidsl = [tidsel] [-e, er] [-e, er, ne] 
traf|truff|træf = traf|truff|træf = [træf] [^a] [^u, ~, et] 
