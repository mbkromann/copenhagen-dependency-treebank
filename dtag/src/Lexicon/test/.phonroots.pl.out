ERROR! cannot read file /opt/dta/arcs.header in Graph->readfile
check that DTAGHOME is set correctly!
ERROR! cannot read file /opt/dta/arcs.trailer in Graph->readfile
check that DTAGHOME is set correctly!
flyv|fløj = flyv|fløj = [flyv] [^ø, v/j] [^ø, v/j, et] 
lav = lav = [lav] [er] [er, r/s] 
lyt|lytt = lyt|lytt = [lyt] [~, er] [~, ede] 
mand|mænd = mand|mænd = [mand] [^æ] [en] 
pukkel|pukl = pukkel|pukl = [pukkel] [-~e, er] [-~e, er, ne] 
tal|tæl = tal|tæl = [tæl] [^a, te] [^a, t] 
tidsel|tidsl = tidsel|tidsl = [tidsel] [-e, er] [-e, er, ne] 
traf|truff|træf = traf|truff|træf = [træf] [^a] [^u, ~, et] 
